app/controllers/article_controller.rb

```diff
@@ -186,4 +186,8 @@ def setup_eds_session(session)
       debug: Settings.EDS_DEBUG
     ).session_token
   end
+
+  def has_search_parameters?
```

Comment:
@camillevilla hmm, what if you add `helper_method :has_search_parameters?` beneath the method definition in the controller and remove the helper?  That might get it working.